packages/test-project/src/components/NestedB.vue

```diff
@@ -0,0 +1,34 @@
+<template lang="html">
+  <div class="">
+    <div style="margin-bottom: 20px">
+      <label>Number B</label>
+      <input
+        v-model.number="numberB"
+        type="number"
+      >
+    </div>
+    <!-- <NestedA /> -->
+    <!-- <pre style="background-color: white;">{{ $v.$errors }}</pre> -->
+  </div>
+</template>
+
+<script>
+import { ref, toRefs } from 'vue'
+import useVuelidate from '@vuelidate/core/src'
+import { required, maxValue, minValue } from '@vuelidate/validators/src/withMessages'
+
+export default {
+  name: 'NestedB',
+  setup (props) {
+    const { max, min } = toRefs(props)
```

Comment:
This is the only way I managed to make this work.

Reply:
I think this is the recommended way.